fvm.md

```diff
@@ -0,0 +1,173 @@
+<img src="https://bafybeiesyiylnbxe5lrim35mwcwckoipn6eg32kpn6nibfdgb4vkmvzsta.ipfs.w3s.link/FVM%20logo.png" width="500" >
```

Comment:
Let's add a mention that it was copied from https://github.com/filecoin-project/community/discussions/585 and give correct attribution.